acquamanCommon.pri

```diff
@@ -3,7 +3,7 @@
 # ####################################################################
 
 # Video Support: Remove this line if you do not have the multimedia module from QtMobility
-CONFIG *= mobility
+#CONFIG *= mobility
```

Comment:
This will have a negative effect on all the other applications.  Also, even though you comment it out here, you still do `CONFIG -= mobility` in your pro file.